Makefile

```
@@ -154,12 +155,17 @@ tidy: fmt $(TIDY_SPELLING) $(TIDY_SHELL)
 
 generate: ; $(info $(M) running go:generate…)
```

checkmake (0.2.2)

[warning] Target body for "generate" exceeds allowed length of 5 (7).

(maxbodylength)